saleor/webhook/payloads.py

```diff
@@ -251,6 +275,21 @@ def generate_checkout_payload(checkout: "Checkout"):
     shipping_method_fields = ("name", "type", "currency", "price_amount")
     lines_dict_data = serialize_checkout_lines(checkout)
 
+    included_taxes_in_price = include_taxes_in_prices()
+
+    voucher_amount = get_voucher_amount(checkout, included_taxes_in_price)
+
+    subtotal = (
```

Comment:
TaxedMoney. You can use Order as an example.

Reply:
Also, we need to cover somehow the case, when we don't have calculated taxes yet.  and we wont to pass some values to these fields